buildpack/datadog.py

```diff
@@ -465,6 +480,9 @@ def stage(buildpack_path, build_path, cache_path):
     logging.debug("Staging Datadog...")
     _download(build_path, cache_path)
 
+    logging.debug("Setting Datadog Agent script permissions if required...")
+    _set_executable("{}/*.sh".format(AGENT_DIR))
```

Comment:
This fix works with Docker Buildpack - but I had to change this line to `_set_executable(os.path.join(build_path, NAMESPACE, "lib", "*.sh"))`.